<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Template Course Creation</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>341</width>
      <height>165</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="courseNameLabel">
         <property name="text">
          <string>Course Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="courseNumberLabel">
         <property name="text">
          <string>Course Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="courseSectionLabel">
         <property name="text">
          <string>Course Section</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="courseSemesterLabel">
         <property name="text">
          <string>Course Semester</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="attendancePointsLabel">
         <property name="text">
          <string>Attendance Points</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="courseNameField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="courseNumberField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="courseSectionField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="courseSemesterField"/>
       </item>
       <item>
        <widget class="QLineEdit" name="attendancePointsField"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="courseChoiceBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>360</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createCourseButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Course</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
